src/Rules/Test.hs

```diff
@@ -22,6 +22,8 @@ testRules = do
         needBuilder $ Ghc CompileHs Stage2
         needBuilder $ GhcPkg Update Stage1
         needBuilder Hpc
+        needBuilder Hsc2Hs
+        need ["inplace/bin/hp2ps"]
```

Comment:
Can we make this less fragile and compute the path? Alternatively, add a TODO to address this later.